packages/cloud-cognitive/src/components/Tearsheet/TearsheetNarrow.stories.js

```diff
@@ -13,7 +13,9 @@ import { pkg } from '../../settings';
 
 import { Button, Form, FormGroup, TextInput } from 'carbon-components-react';
 
-import { Tearsheet, TearsheetNarrow } from '.';
+import { Tearsheet } from '.';
+import { TearsheetNarrow, deprecatedProps } from '.';
```

Comment:
This seems like the problem. Should be imported like this

```js
import { TearsheetNarrow, deprecatedProps } from './TearsheetNarrow';
```

Reply:
Ah, unchecked in change. Good catch.